### DISCUSSION TITLE
Wishlist for index management

### DISCUSSION THREAD

**Discussion by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#1575] [Ideas]
This is to keep track of a discussion we're having. We're dealing with two issues:

- So far, what corpus is indexed when is mostly managed through verbal agreement. However, I-analyzer is growing and we have the ambition to add many more corpora. We need a proper "system" to keep track of elasticsearch indices.
- We're working on adding corpora without defining them in the source code. Related to that, it would make sense if you  can also create the index from the interface and have some level of control over indices (#1007).

In other words, we need some functionality for tracking and managing indexing, probably through the Django admin. Before we make concrete plans for this: what would an ideal system for index management look like?

- What metadata about an index would be insightful on the server where you're running the index job?
- What metadata is insightful on a server that uses the index passively (but does not write to it)?
- What actions should be supported?

**Comment by lukavdplas** [comment]
I think this discussion is obsolete, since we now have a system to manage indexing in the admin site. For any further issues or feature requests, create a new issue or discussion.
